xbmc/video/VideoInfoTag.cpp

```diff
@@ -820,3 +869,168 @@ unsigned int CVideoInfoTag::GetDurationFromMinuteString(const std::string &runti
   }
   return duration*60;
 }
+
+void CVideoInfoTag::SetBasePath(const std::string &basePath)
+{
+  m_basePath = Trim(basePath);
+}
+
+void CVideoInfoTag::SetDirector(const std::vector<std::string> &director)
```

Comment:
Make this non const and && otherwise std::move does nothing and we still copy the array, same for the other methods that take an array